src/puptoo/process/profile.py

```diff
@@ -5,7 +5,9 @@
 
 from insights import make_metadata, rule, run
 from insights.combiners.cloud_provider import CloudProvider
+from insights.combiners.cloud_instance import CloudInstance
 from insights.combiners.redhat_release import RedHatRelease
+from insights.combiners.os_release import OSRelease
```

Comment:
@xiangce Is import `OSRelease` required here? As it is not used anywhere.

Reply:
Sorry for this, I just removed it in the latest commit, please have a look.